cfme/infrastructure/provider/__init__.py

```diff
@@ -167,7 +167,7 @@ def vm_name(self):
         return version.pick({
             version.LOWEST: "VMs",
             '5.5': "VMs and Instances",
-            '5.7': "Virtual Machines"})
+            '5.7.1': "Virtual Machines"})
```

Comment:
I changed the version for new string to 5.7.1, because 5.7.0 still has the old string.
Why would we remove support for 5.5? We might still need to test 5.5, and I don't see any reason to intentionally remove the support.